Dockerfile-quickstart

```diff
@@ -0,0 +1,18 @@
+FROM sugarush/cayley
```

Comment:
It's a good idea to mirror an image to `quay.io/cayleygraph/cayley`, thus this `FROM` statement might be different.